build.assets/Makefile

```diff
@@ -509,7 +509,7 @@ build-planet-sub:
 	if [ ! -d "$(PLANET_SRCDIR)/planet" ]; then cd $(PLANET_SRCDIR) && git clone $(PLANET_REPO); fi
 	cd $(PLANET_SRCDIR)/planet && git fetch --all && git checkout $(PLANET_BRANCH)
 	@echo -e "\n----> Building Planet images...\n"
-	BUILDDIR=$(PLANET_BUILDDIR) make -C $(PLANET_SRCDIR)/planet production USER=jenkins
+	BUILDDIR=$(PLANET_BUILDDIR) make -C $(PLANET_SRCDIR)/planet production
```

Comment:
This is safe -- see https://github.com/gravitational/planet/blob/96adc933585be4ab6f7e90b64bf7ed9c4d70c01c/Makefile#L223 for the only place this variable is used.  In this case, falling back to the shell's $USER is a better option than hardcoding `jenkins` which would have only worked in our old build environment.